spec/requests/posts_controller_spec.rb

```diff
@@ -4,6 +4,8 @@
 
 describe DiscourseJira::PostsController do
   fab!(:admin)
+  fab!(:non_admin_user_with_access) { Fabricate(:user) } # stuff
+  fab!(:joe_schmoe) { Fabricate(:user) } # stuff
```

Comment:
I don't think you intended to leave these comments here, right?

Reply:
HAHA yes I meant to delete those 😭